### DISCUSSION TITLE
Welcome to Sakura-13B-Galgame Discussions!

### DISCUSSION THREAD

**Discussion by sakura-umi** [SakuraLLM/SakuraLLM#50] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tukangcode** [comment]
hello i tukangocoder, i have some question regarding license use, what license this model applied ? are it allows comercila use ? thanks.

**Reply by CberYellowstone** [reply]
CC BY-NC-SA 4.0

**Reply by sakura-umi** [reply]
all models [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) and no use for commercial purposes.
